server/src/main/java/org/apache/druid/server/router/CoordinatorRuleManager.java

```diff
@@ -149,7 +148,7 @@ public void poll()
         );
       }
 
-      ConcurrentHashMap<String, List<Rule>> newRules = new ConcurrentHashMap<>(
+      Map<String, List<Rule>> newRules = new HashMap<>(
```

Comment:
`new HashMap()` looks unnecessary.

Reply:
do you mean set rules directly, no need create newRules?